azure-pipelines.yml

```diff
@@ -88,7 +88,7 @@ stages:
       dependsOn: HashBaseDependencies
       imageName: data61/anonlink-app
       # this part is still todo automated
-      dockerBuildArgs: "--build-arg VERSION=02a973ce02c96d24376c5770c0426ccb0958a0d59136a4f2fc11c8666c43062a"
+      dockerBuildArgs: "--build-arg VERSION=ed0be6ee130ec505b2a65af26ba570e1033ffb7e0cc6b27b6a8520df94f65d20"
```

Comment:
this is really annoying. When we change the base image requirements, although a new version gets build, it will not be tested. We already had a non-functioning base image. We really need to find a way to test the new base images straight away.